packages/@okta/vuepress-site/docs/api/resources/oidc/index.md

```diff
@@ -1338,33 +1226,74 @@ For more information about configuring an app for OpenID Connect, including grou
+When registering an OAuth 2 client application, you can specify an authentication method for the `/token` endpoint by including the [token_endpoint_auth_method](https://developer.okta.com/docs/api/resources/apps/#add-oauth-2-0-client-application) parameter. Okta supports five token authentication methods:
 
-If you configure your client to use a `client_secret` [Client Authentication Method](http://openid.net/specs/openid-connect-core-1_0.html#ClientAuthentication), provide the [`client_id` and `client_secret`](#request-parameters-1)
-using either of the following methods:
+> Note: You can use only one of these methods in a single request or an error occurs.
 
-* Provide `client_id` and `client_secret`
-  in an Authorization header in the Basic auth scheme (`client_secret_basic`). For authentication with Basic auth, an HTTP header with the following format must be provided with the POST request:
+* `client_secret_basic`: Use this method when blah blah blah
+* `client_secret_post`: Use this method when blah blah blah
+* `client_secret_jwt`: Use this method in the following use cases:
+  * User delegation grant such as `grant_type=authorization_code` or `grant_type=password`
+  * Client authentication and authorization for a client acting as itself with `grant_type=client_credentials`
+  * Client authentication for token introspection and token revocation
+* `private_key_jwt`: Use this method when you need to enable the authorization server to validate that it is your application making the call by verifying the signed token.
+* `none` - Use this method when the client is a public client and doesn't have a client secret.
+
+> Note: If you don't specify a method when registering your client, the default method is `client_secret_basic`.
+
+### Client Secret Client Authentication Methods
+If you configured your client to use a `client_secret` [client authentication method](http://openid.net/specs/openid-connect-core-1_0.html#ClientAuthentication), provide the `client_id` and `client_secret` using one of these methods: 
+
+> Note: The first two methods are the most common scenarios.
+
+* `client_secret_basic`: Provide the `client_id` and `client_secret` values in the Authorization header as a Basic auth base64-encoded string with the POST request:
   ```bash
-  Authorization: Basic ${Base64(<client_id>:<client_secret>)}
+    Authorization: Basic ${Base64(<client_id>:<client_secret>)}
+    ```
+* `client_secret_post`: Provide the `client_id` and `client_secret` as additional parameters in the POST request body.
+
+* `client_secret_jwt`: Provide the `client_id` in a JWT that you sign with the `client_secret` using an HMAC SHA algorithms (HS256, HS384, or HS512). The JWT contains other values, such as issuer and subject and can also contain custom claims. See [Token Claims for Client Authentication with Client Secret or Private Key JWT](/docs/api/resources/oidc/#token-claims-for-client-authentication-with-client-secret-or-private-key-jwt). 
```

Comment:
I think that saying that it "can also contain custom claims" is unnecessary here